srcalc/src/test/java/gov/va/med/srcalc/domain/SampleObjects.java

```diff
@@ -286,4 +286,17 @@ public static Expression expression2()
         final SpelExpressionParser parser = new SpelExpressionParser();
         return parser.parseExpression("false");
     }
+    
+    public static Calculation calculatedCalculation() throws Exception
```

Comment:
I think this method is complicated enough to warrant a Javadoc. (What do you think about a policy of every method needs a Javadoc?)